provider/resources.go

```diff
@@ -3,14 +3,17 @@
 package gcp
 
 import (
+	"context"
 	"fmt"
+	"os"
 	"path/filepath"
 	"strings"
 	"unicode"
 
 	google "github.com/hashicorp/terraform-provider-google-beta/google-beta"
 	"github.com/pulumi/pulumi-gcp/provider/v6/pkg/version"
 	"github.com/pulumi/pulumi-terraform-bridge/v3/pkg/tfbridge"
+	shim "github.com/pulumi/pulumi-terraform-bridge/v3/pkg/tfshim"
```

Comment:
I just wanted to make sure that it's intended to use both v1 and v2 versions of `tfshim`?

Reply:
yeah - the preConfigureCallback takes a shim :/